### DISCUSSION TITLE
doesnt work!!!!!

### DISCUSSION THREAD

**Discussion by MuSinepValutab** [zeusssz/hercules-obfuscator#32] [General]
i have lua, script and obfuscator like i need it, but cmd refuses to work
maybe i cant read tbh but plz help

![image](https://github.com/user-attachments/assets/deb143d5-9444-49c8-9f7c-4c7cc19a0369)

**Comment by MuSinepValutab** [comment]
opening lua src\hercules.lua just shows the lua file btw so the file is in the right spot

**Comment by zeusssz** [comment]
you shouldve made an issue. this is because the `hercules-obfuscator-main` dir is the current directory, while the file you are obfuscating is not in that directory range, rather, it is outside of it, being `~\Desktop\skript.lua`, while your `cd` is `~\Desktop\src\`

also, i reccommend using the lua compiler

**Reply by zeusssz** [reply]
also, use `hercules.lua` instead of `hercules`
